src/Microsoft.Tye.Hosting/Watch/assets/DotNetWatch.targets

```diff
@@ -26,6 +26,7 @@ Returns: @(Watch)
     </_CollectWatchItemsDependsOn>
     <_CollectWatchItemsDependsOn Condition=" '$(TargetFramework)' != '' ">
       _CoreCollectWatchItems;
+      $(CustomCollectWatchItems);
```

Comment:
@pranavkm we should potentially talk about this; we copied the dotnet watch code into tye, and made a few modifications to instead run the executable rather than calling `dotnet run`. Is there any unification we could consider here?

Reply:
VS scans for dotnet-watch assets in the SDK. Is this something we could do with Tye? Alternatively we could always ship a sources packages to make sure we're in sync.